src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/BurdenEstimatesRouteConfig.kt

```diff
@@ -47,6 +47,10 @@ object BurdenEstimatesRouteConfig : RouteConfig
                     .json()
                     .secure(writePermissions),
 
+            Endpoint("$baseUrl/:set-id/actions/upload/:token/", uploadController, "uploadBurdenEstimateFile", method = HttpMethod.post)
+                    .json()
+                    .secure(writePermissions),
+
             Endpoint("$baseUrl/:set-id/", uploadController, "populateBurdenEstimateSet", method = HttpMethod.post)
```

Comment:
Is the old endpoint going to be removed eventually, or left in place? If we're leaving it, we maybe should have some way of clearly distinguishing it as a different sort of beast from the endpoints to do with chunked file upload. This is already implied because it has a different path (not part of /actions). Maybe just a comment?

Reply:
No, the old endpoints will still be useful for programming against. Source of truth should be the spec, which alerted me to the fact I hadn't added the new endpoints to the spec! Added them now.